hacking/aws_config/setup-iam.yml

```diff
@@ -1,55 +1,97 @@
-# Usage: ansible-playbook setup-iam.yml -e iam_group=ansible_test -vv
+# Usage: ansible-playbook setup-iam.yml -vv
 #
-# Creates IAM policies and associates them with iam_group. This group
-# can then be associated with an appropriate user
+# Creates IAM policies and associates them with two iam_groups. These groups
+# can then be associated with an appropriate user:
 #
-# You can pass -e profile=boto_profile_name if you have a profile that
-# you can use, otherwise use normal AWS methods (env variables, instance
-# profile, etc)
+# Groups:
+# - ansible-integration-ci
+#   - Mirrors the permissions used by the the AWS collections CI
+# - ansible-integrayion-unsupported
+#   - Additional Permissions on top of the 'CI' permissions
+#     required to run the 'unsupported' tests
 #
-# If you want to use a region other than us-east-1 (and only us-east-2
-# works with ansible-test), pass -e region=us-east-2
+# Policies:
+# - AnsibleIntegration-CI-*
+#   - Managed Policies used by the {{ iam_group_prefix }}-ci group
+# - AnsibleIntegration-Unsupported-*
+#   - Managed Policies used by the {{ iam_group_prefix }}-unsupported group
 #
-# Requires 2.4 for iam_managed_policy and iam_group
+# Supported variables:
+# - region: The AWS region that many of the policies will be restricted to
```

Comment:
I'd like to be clearer here than "many".  Are there policies that are hard coded to some region, or could this be said as "The AWS region that any region-restricted policies will be limited to"?